File: app/api/schema.py
schema = [
    {
        "entityType" : "Gene",
        "entityIdKey" : "geneId",
        "entityNameKey" : "geneName",
        "properties" : [
            "geneDPI",
            "geneDSI",
            "geneId",
            "genePLI"
        ],
        "relationships" : [
            {
                "NeighborType" : "Disease",
                "NeighborNameKey" : "diseaseName",
                "RelationName" : "AssociatesWith",
                "FromNode" : "Gene"
            }
        ]
    },
    {
        "entityType" : "Disease",
        "entityIdKey" : "diseaseId",
        "entityNameKey" : "diseaseName",
        "properties" : [
            "diseaseType"
        ],
        "relationships" : [
            {
                "NeighborType" : "Gene",
                "NeighborNameKey" : "geneName",
                "RelationName" : "AssociatesWith",
                "FromNode" : "Gene"
            }
        ]
    }
]

File: app/client-app/src/components/schema.js
const schema = [
    {
        entityType: "gene",
        propertiesOfImportance: [
            "geneName", "geneId"
        ],
        nameKey: "geneName"
    },
    {
        entityType: "disease",
        propertiesOfImportance: [
            "diseaseName", "diseaseId"
        ],
        nameKey: "diseaseName"
    }
]

export default schema